.circleci/docker/php5.5/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:14.04
```

Comment:
Is there a circle convenience image we could base off of? It gives us a lot of advantages over plain ubuntu

Reply:
CircleCI doesn't have any convenience images for php5.5, so no.

Reply:
Then let's base off of `cimg/base:2020.01`

Reply:
Let me give it a try. But I don't know if it's going to work since PHP5.5 is long deprecated and I'm pretty sure apt-get won't work.

Reply:
@roopakv Yup as I suspected. It just straight up doesn't work. Package 'php5' has no installation candidate

Reply:
aight that makes sense. When we need the tools ill try add them.